Sources/NIO/EventLoop.swift

```diff
@@ -622,6 +626,9 @@ public protocol EventLoopGroup: class {
     /// The virtue of this function is to shut the event loop down. To work around that we call back on a DispatchQueue
     /// instead.
     func shutdownGracefully(queue: DispatchQueue, _ callback: @escaping (Error?) -> Void)
+
+    /// `EventLoop`s of the group
```

Comment:
Mind documenting this a bit more extensively? What it's here for, what the rules are for it, why it's optional, that sort of thing?